static/js/components/EducationDisplay.js

```diff
@@ -55,23 +53,27 @@ export default class EducationDisplay extends ProfileFormFields {
       level.value === entry.degree_name
     )).label;
     let icons = () => (
-      <Cell col={2} className="profile-row-icons">
+      <div className="profile-row-icons">
         {validationAlert()}
         <IconButton className="edit-button" name="edit" onClick={editEntry} />
         <IconButton className="delete-button" name="delete" onClick={deleteEntry} />
-      </Cell>
+      </div>
     );
     return (
-      <Grid className="profile-tab-card-grid user-page" key={index}>
-        <Cell col={4} className="profile-row-name">
-          <span className="school-type">{ degree }</span><br/>
-          { entry.school_name }
-        </Cell>
-        <Cell col={6} className="profile-row-date-range">
-          {`${dateFormat(entry.graduation_date)}`}
-        </Cell>
+      <Cell col={12} className="profile-form-row" key={index}>
+        <div>
```

Comment:
Instead of using `first-child` in the CSS can you add a class to this div so it's more descriptive what it contains?